client/js/app/utils/ExplorerUtils.js

```diff
@@ -79,36 +82,34 @@ module.exports = {
     }
     var params = _.cloneDeep(explorer.query);
 
-    // Set the timeframe (will get removed if it's null o undefined)
-    params.timeframe = module.exports.getTimeframe(explorer);
```

Comment:
We could potentially even just do a time `if (params.time)` without even really worrying out the specific analysis type.